### PR TITLE
Several fixes to the spec

### PR DESCRIPTION
**Pull request by Nutlope**
# Changes made
- Fix file upload types: `FileResponse`, `FileList`, and `FileObject`
- Fix fine-tuning types: `FinetuneList`, and `FinetuneListEvents`
- Added min_p, presence penalty, freq penalty, and logit_bias
- Added examples for embeddings/images

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -630,6 +637,30 @@ components:
...
+          description: The `logit_bias` parameter allows us to adjust the likelihood of specific tokens appearing in the generated output.
+          properties:
+            token_index:
+              type: number
+              format: int32
+              description: The index of the token to bias.
+            bias:
+              type: number
+              format: float
+              description: The logit bias value to apply to the token.
```
Comment (orangetin) [inline openapi.yaml:663]: logit_bias is a dictionary with string keys and float values. the keys are the token indices and the values are the bias. It looks like this:
```
logit_bias: {
    "1314": -10.5,
     "2042": 20.7
}
```

File: openapi.yaml [hunk 2]
```diff
@@ -802,6 +833,30 @@ components:
...
```
Comment (orangetin) [inline openapi.yaml:859]: same issue here with logit bias as in previous comment

**Review by orangetin — COMMENTED** [review]
logit_bias has the wrong type. rest looks good!

File: openapi.yaml (hunk 1 above)
Comment (Nutlope) [reply openapi.yaml]: Good callout, fixed!

**Review by orangetin — APPROVED** [review]

**Pull request merged**